ratatool-diffy/diffy.md

```diff
@@ -0,0 +1,31 @@
+Diffy
+=======
+
+Diffy contains record diff-ing classes that can be utilized by BigDiffy to perform Diffs over large datasets.
+
+# BigDiffy
+
+BigDiffy will run a [Scio](https://github.com/spotify/scio) pipeline diffing a LHS and RHS by key
+ and output statistics based on the differences. The output contains 3 folders:
+
+ * `global` - Global counts of Diff types (`SAME`, `DIFFERENT`, `MISSING_LHS`, `MISSING_RHS`) seen in the entire dataset
+ * `fields` - Field level statistics including, but not limited to, the number of records with different values per field, min, max, standard deviation 
+ * `keys` - All unique keys found in the two datasets and their Diff types by key
+
+For full details see [BigDiffy.scala](https://github.com/spotify/ratatool/blob/master/ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala)
+
+## Usage
+
+```
```

Comment:
do we want to discuss the output and how to read it as well

Reply:
It's discussed a bit in the earlier section. Do you think it needs to be more detailed?

Reply:
Yeah maybe give an example of how to read the output since I know we mentioned it's hard to parse potentially